### PR TITLE
[web] Added contribution guide for web

### PR DESCRIPTION
**Pull request by EvanBacon**
# Why

Moved from https://github.com/expo/web-examples

### CODE REVIEW

**Comment by amandeepmittal** [comment]
@EvanBacon there are a couple of typos, would you mind if I amend them? 😄

File: docs/pages/versions/unversioned/guides/contributing-to-web.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: Contributing to Web
+---
+
+There are a couple different things you could work on in regards to Expo for web. Here we'll cover how you can setup your environment to easily contribute.
+
+## 💎 Features & Tests
+
+To work on modules like `expo-camera` or even community libraries, I like to work in the expo monorepo, this is because it's a yarn workspace which links the packages really well.
```
Comment (ide) [inline docs/pages/versions/unversioned/guides/contributing-to-web.md]: Let's change the tone from casual & first-person to more second-person like the rest of our docs.

File: docs/pages/versions/unversioned/guides/contributing-to-web.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
+title: Contributing to Web
+---
+
+There are a couple different things you could work on in regards to Expo for web. Here we'll cover how you can setup your environment to easily contribute.
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/guides/contributing-to-web.md]: There are a couple of different things you could work on in regards to Expo for web. Here we'll cover how you can setup your environment to easily contribute.

File: docs/pages/versions/unversioned/guides/contributing-to-web.md (hunk 1 above)
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/guides/contributing-to-web.md]: To work on modules like `expo-camera` or community libraries, consider working in the expo monorepo. The reason for this is that the monorepo is a yarn workspace which links the packages really well.

File: docs/pages/versions/unversioned/guides/contributing-to-web.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- Clone the repo `git clone git@github.com:expo/expo.git`
+- [Set up the project](https://github.com/expo/expo/#set-up).
+- I also tend to run `git submodule init && git submodule update && git submodule foreach --recursive git checkout . && git lfs pull` just to be safe! 😄
+
+Now to develop Unimodules, work on apps in the `expo/apps` folder.
+These will use the modules in the `expo/packages` folder, meaning you don't need to worry about accidentially erasing changes by working in `node_modules` 😬.
+
+### Working on modules
+
+For example if you want to work on `react-navigation`, clone it to `expo/packages/react-navigation` and run `yarn`, now projects like `expo/apps/native-component-list` will use your local copy of navigation and update when you make changes.
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/guides/contributing-to-web.md]: For example, if you want to work on `react-navigation`, clone it to `expo/packages/react-navigation` and run `yarn`. Now projects like `expo/apps/native-component-list` will use your local copy of navigation and update when you make changes.

File: docs/pages/versions/unversioned/guides/contributing-to-web.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+### Best Practices
+
+For absolute best practices, I use the `expo-module-scripts` which unify the TypeScript configuration and make testing really easy. The setup looks something like this:
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/guides/contributing-to-web.md]: For absolute best practices, use the `expo-module-scripts` that unifies the TypeScript configuration and makes testing really easy. The setup looks something like this:

File: docs/pages/versions/unversioned/guides/contributing-to-web.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+- Run `yarn start` in the root of the `expo-cli/` repo (have this running while making changes).
+- Add an alias on your computer by opening the bash profile (`code ~/.bash_profile`).
+- Add the line `alias expod="/Users/evanbacon/Documents/GitHub/expo-cli/packages/expo-cli/bin/expo.js"` but with your local `expo-cli` path.
+- Restart the terminal.
+- Now when you want to use debug features run `expod` instead of `expo`.
+
+## 📚 Docs & Guides
+
+Currently we are temporarily adding documentations to this repo (`web-examples`). In the future we will move this to [Expo Docs](https://docs.expo.io). Until then, you can open issues or pull requests with new documentation, ideas, or questions here! 😁
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/guides/contributing-to-web.md]: Currently, we are temporarily adding documentation to this repo (`web-examples`). In the future, we will move this to [Expo Docs](https://docs.expo.io). Until then, you can open issues or pull requests with new documentation, ideas, or questions here! 😁

**Review by amandeepmittal — CHANGES_REQUESTED** [review]
@EvanBacon I could not edit the file directory, so I am submitting some changes I think to fix the typos and improve the overall tone, as suggestions.

**Pull request closed without merging**